website/docs/usage/jujutsu.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 12
+---
+
+# Jujutsu
+
+You can use with a repository that has been cloned using [jujutsu](https://martinvonz.github.io/jj/latest/).
+
+## Colocated
+
+If the repository was cloned by `jujutsu` using the `--colocate` option, then all you need to do is make sure that
+you have checked out your mainline branch using git.
+If you don't, then you will likely see an error about an unborn branch.
+
+## Non-colocated
+
+If the repository was cloned by `jujutsu` but *not* using the `--colocate` option,
+then the Git repository, normally the `.git` directory, is located in `.jj/repo/store/git`
+
+You can do one of two things to make your repository compatible with `git-cliff`:
+
+### Fake Colocation
+
+Create a file in the root of your repository that tells Git, and `git-cliff` where the Git repository is:
+
+```bash
+echo "gitdir: .jj/repo/store/git" > .git
+```
```

Comment:
I tried this for a cloned repository but it didn't work for me:

```
$ jj git clone https://github.com/orhun/menyoki
$ echo "gitdir: .jj/repo/store/git" > .git

# and then run git-cliff built from this branch
#     Running `/drv/cargo/target/debug/git-cliff -r /tmp/tmp.sP4i0aaZ62/menyoki -c config/cliff.toml`
 ERROR git_cliff > Git error: `reference 'refs/heads/master' not found; class=Reference (4); code=UnbornBranch (-9)`
```

Reply:
Okay. @orhun: how about this:

```bash
jj git clone https://github.com/orhun/menyoki
cd menyoki
echo "gitdir: .jj/repo/store/git" > .git
echo "ref: refs/remotes/origin/master" > .jj/repo/store/git/HEAD
```

![2024-10-29T07-48-04Z_code](https://github.com/user-attachments/assets/7554836b-c074-4f77-ac4b-cad4b9724848)

Reply:
Yup, that works. In that case we shouldn't present it as "you need to do one of these two options" in the documentation then.

Reply:
lol yeah. I've merged them.